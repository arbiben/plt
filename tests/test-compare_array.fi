int main(){
arr int 6 a?
arr int 6 b?

a = [1,2,3,4,5,6]?
b = [1,2,3,4,5,6]?

if (a@1 is b@1){
printstring("a@1 is b@1!")?
}
}
