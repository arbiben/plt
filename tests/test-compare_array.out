a@1 is b@1!
